report/src/sections/04-theory/02-abstract-semantics.tex

```diff
@@ -72,7 +71,7 @@ \subsubsection{Abstract Semantics of Boolean expressions}
 
 \begin{table}[H]
     \centering
-    \caption{test}
+    \caption{}
```

Comment:
Empty captions? Hvis det er equations som skal alignes men uden nummer så brug align* i stedet for table+tabular. Samme effekt.